src/datalad_installer.py

```diff
@@ -1076,6 +1076,7 @@ def provide(
                 conda_instance.conda_exe,
                 "update",
                 "-n",
+                "-y",
```

Comment:
note: oh -- it was solely for upgrading the buggy version of it, that is why not generally reproducible, gotcha